tests/create_task_output_wildcard.yml

```diff
@@ -0,0 +1,50 @@
+name: Create Tes Task Job with Wildcard Outputs
+description: Job to create a new TES Task using glob patterns in outputs
+service: TES
+compatible_versions:
+  - v1.1
+tags:
+  - logical
+  - create task
+  - create task test
+  - all
+jobs:
+  - name: create_task
+    description: Create a new TES task
+    endpoint: /tasks
+    operation: POST
+    request_body: |
+      {
+        "name": "CompTest",
+        "description": "CompTest",
+        "executors": [
+          {
+            "image": "ubuntu:20.04",
+            "command": [
+              "TestString1",
+              "TestString2"
+            ]
+          },
+          {
+            "image": "ubuntu:20.04",
+            "command": [
+              "TestString3",
+              "TestString4"
+            ]
+          }
+        ],
+        "outputs": [
+          {
+            "url": "https://my.storage.org/bucket/",
+            "path": "/path/in/container/*/*.?am",
+            "path_prefix": "/path/in/container"
+          },
+          {
+            "url": "https://my.storage.org/bucket/wildc*ard_char?",
+            "path": "/path/with/wildc\*rd_char\?"",
```

Comment:
This test will fail earlier due to an invalid storage URL. Cloud-integrated tests are not supported currently. A proper test would include accessing a storage URL, inputting a file, defining the commands for output files and then verifying the above output path conditions. The issue is already raised for this and maybe a test can be added in future.